plugins/statistics.koplugin/main.lua

```diff
@@ -1314,18 +1313,35 @@ function ReaderStatistics:getCurrentStat(id_book)
         { _("Reading started"), os.date("%Y-%m-%d (%H:%M)", tonumber(first_open))},
         { _("Days reading this book"), tonumber(total_days) },
         { _("Average time per day"), util.secondsToClock(book_read_time/tonumber(total_days), false) },
-        -- per page
-        { _("Pages read"), tonumber(total_read_pages) },
+        -- per page (% read)
         { _("Average time per page"), util.secondsToClock(self.avg_time, false) },
+        { _("Pages read"), string.format("%d (%d%%)", total_read_pages, Math.round(100*total_read_pages/self.data.pages)) },
+        -- current page (% completed)
+        { _("Current page/Total pages"), string.format("%d/%d (%d%%)", self.curr_page, self.data.pages, Math.round(100*self.curr_page/self.data.pages)) },
         -- estimation, from current page to end of book
-        { _("Current page/Total pages"),  self.curr_page .. "/" .. self.data.pages },
-        { _("Percentage completed"), formatstr:format(self.curr_page/self.data.pages * 100) },
-        { _("Estimated time to read"), util.secondsToClock(time_to_read, false) },
-        { _("Estimated reading finished"),
-            T(N_("%1 (1 day)", "%1 (%2 days)", estimate_days_to_read), estimate_end_of_read_date, estimate_days_to_read) },
-
-        { _("Highlights"), tonumber(highlights) },
+        { _("Estimated time to read"), estimates_valid and util.secondsToClock(time_to_read, false) or _("n/a") },
```

Comment:
nit: I tend to prefer `N/A` to `n/a` (but we do have a few n/a instances in the coded base. less that N/As, though ;p).

Reply:
OK, I switched them all to `N/A` (no preference myself :)